### ISSUE TITLE
Default dacfactory extension

### ISSUE THREAD

**Issue by michaeljyeates** [eosdac/eosdac-client-legacy#92]
The current boilerplate is designed to show people the capabilities, we also need a default extension for dacs created with dacfactory

**Comment by piecesnbits** [comment]
@michaeljyeates can't you clone the current boilerplate locally and use this "copy" as default for the factory dacs?

**Comment by michaeljyeates** [comment]
Yeah, we would have 2 extensions.  One boilerplate and one for the factory.  It would need to be in git though because thats how the factory gets it now
